area_detector_handlers/tests/test_xs3.py

```diff
@@ -0,0 +1,39 @@
+import pkg_resources
```

Comment:
The implementation in `pkg_resources` is "problematic", per Min. I think it's something about being slow and maybe bad with eggs. Main reason to care here is just to set a good example / be consistent in using `entrypoints` everywhere.

```suggestion
import entrypoints
```

Reply:
Explanation: https://github.com/takluyver/entrypoints/pull/37/files